.github/workflows/e2e-openshift-4.1.yaml

```diff
@@ -0,0 +1,33 @@
+name: "OpenShift end-to-end tests, ES operator 4.1"
+on: [push, pull_request]
+
+jobs:
+  end-to-end:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        TEST_GROUP: [es-self-provisioned]
+    steps:
+      - uses: jpkrohling/setup-kubectl@v1-release
+      - uses: jpkrohling/setup-operator-sdk@v1-release
+      - uses: actions/checkout@v1
+        with:
+          path: src/github.com/jaegertracing/jaeger-operator # remove when using operator >= 0.9.0
+
+      - name: "setup docker"
+        run: ./.ci/setup-docker.sh
+
+      - name: "set max_map_count"
+        run: sudo sysctl -w vm.max_map_count=262144
+
+      - name: "start openshift"
+        run: ./.ci/start-openshift.sh
+
+      - name: "running end to end test"
+        env:
+          GOPATH: /home/runner/work/jaeger-operator #remove when using operator >= 0.9.0
+          CI: true
+          TEST_GROUP: ${{ matrix.TEST_GROUP }}
+          ES_OPERATOR_BRANCH: release-4.1
+          ES_OPERATOR_IMAGE: quay.io/openshift/origin-elasticsearch-operator:4.1
+        run: ./.ci/run-e2e-tests.sh
```

Comment:
Should it run all e2e tests here, or just the es ones?

Reply:
The original OCP workflow run `es-self-provisioned` test group. This file is just copied to run the same CI as before.

Reply:
@jpkrohling Thoughts - should it just be limited to run the self provisioned tests? and also update the other workflow aswell?

Reply:
@objectiser It seems like a separate issue/discussion. Could you please create an issue to discuss which tests should be run on OCP?

Reply:
> should it just be limited to run the self provisioned tests? and also update the other workflow aswell?

Good question. I think I would still like to know if something is broken when using an externally managed ES. It might have been limited first for development purposes, and I probably forgot to enable it for the other tests.

Reply:
I am might be missing something, but it seems to me we are talking about different things here.

There is a test for externally managed elasticsearch - the group name is `es`. The question is whether we want to run it on openshift, at the moment we run it only on k8s.

I think externally managed elasticsearch test on openshift is not very important. However there might be other tests more suitable to run on Openshift e.g. test oauth proxy (I am not sure if there is such a test).

Reply:
Ok will open a separate issue.